android/app/build.gradle

```diff
@@ -0,0 +1,66 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "org.freedesktop.gstreamer.play"
+        minSdkVersion 19
+        targetSdkVersion 22
+
+        ndk {
+            moduleName "gstplayer"
+        }
+    }
+
+    sourceSets {
+        main {
+            // GStreamer will generate these files.
+            java {
+                srcDir 'src/main/jni/src'
+            }
+            assets {
+                srcDir 'src/main/jni/assets'
+            }
+
+            //Tell Gradle where to put the compiled shared library
+            jniLibs.srcDir 'src/main/libs'
+
+            //disable automatic ndk-build call
+            jni.srcDirs = [];
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+
+    // Thank you http://stackoverflow.com/q/28878689/375209
+    tasks.withType(JavaCompile) {
+        compileTask -> compileTask.dependsOn ndkBuild
+    }
+
+    task ndkBuild(type: Exec) {
+        Properties properties = new Properties()
+        properties.load(project.rootProject.file('local.properties').newDataInputStream())
+        def ndkDir = properties.getProperty('ndk.dir')
+        environment GSTREAMER_SDK_ROOT_ANDROID: properties.getProperty('gstreamerSdk.dir')
```

Comment:
The variable names are supposed to be GSTREAMER_ROOT_ANDROID, without the SDK part :) And I guess as such it should also be gstreamer.dir and not gstreamerSdk.dir